src/pages/events/configure-commerce.md

```diff
@@ -0,0 +1,193 @@
+   php ./vendor/bin/ece-tools run scenario/build/generate.xml
+   php ./vendor/bin/ece-tools run scenario/build/transfer.xml
+```
+
+This hook generates and enables the `AdobeCommerceEvents` module, which allows you to register custom events.
+
+<!--Note to reviewer: I deleted references of bin/magento events:metadata:populate because I'm going to assume that a custom module with specific pre-defined events isn't available at initial installation/configuration. I will create a separate "Module development/integration" topic that addresses this command. -->
+
+### Local and on-premises installation
+
+1. Update the project dependencies.
+
+  ```bash
+  composer update
+  ```
+
+1. Run the following command to generate the `AdobeCommerceEvents` module. This module helps register  events. At this point, the module is empty.
+
+   ```bash
+   bin/magento events:generate:module
+   ```
+
+1. Enable the newly-installed modules:
+
+   ```bash
+   bin/magento module:enable --all
+   ```
+
+1. Upgrade your instance:
+
+   ```bash
+   bin/magento setup:upgrade
+   ```
+
+1. Compile your instance to generate new classes:
+
+   ```bash
+   bin/magento setup:di:compile
+   ```
+
+## Begin configuring events on Commerce
+
+You must configure Commerce to communicate with your project. You will need two files that you downloaded from the the Adobe Developer Console.
+
+1. In the Commerce Admin, navigate to **Stores** > Settings > **Configuration** > **Adobe Services** > **Adobe I/O Events** > **General configuration**.
+
+1. Copy and paste the contents of the `private.key` file into the **Service Account Private Key** field. Use the following command to copy the contents.
+
+   ```bash
+   cat config/private.key | pbcopy
+   ```
+
+1. Copy the contents of the `<workspace-name>.json` file into the **Adobe I/O Workspace Configuration** field.
+
+1. Enter a unique identifier in the **Adobe Commerce Instance ID** field. This value can be any unique string.
+
+1. Click **Save Config**, but do not leave the page. The next section creates an event provider, which is necessary to complete the configuration.
+
+## Create an event provider and finish Commerce configuration
+
+You cannot create an event provider until you have configured and saved a private key, workspace file, and instance ID values.
+
+1. Create a `<Commerce-root-directory>/app/etc/event-types.json` file and add the following:
+
+   ```json
+   {
+    "provider": {
+        "label": "My Adobe Commerce Events",
+        "description": "Provides out-of-process extensibility for Adobe Commerce"
+        }
+    }
+    ```
+
+1. Run the following command to create an event provider:
+
+   ```bash
+   bin/magento events:create-event-provider
+   ```
+
+   The command displays a message similar to the the following:
+
+   ```terminal
+   No event provider found, a new event provider will be created
+   A new event provider has been created with ID 63a1f8fe-e911-45a4-9d3f
+   ```
+
```

Comment:
As PR https://github.com/magento-commerce/module-adobe-io-events/pull/3 was merged now we can simplify and remove step 1 and leave only the next command: 
```
bin/magento events:create-event-provider --label='Provider label' --description='Provider description'
```